tools/integration-tests/tester/tests/mana/mana_test.go

```diff
@@ -201,7 +203,7 @@ func TestManaApis(t *testing.T) {
 
 	// Test /mana/access/nhighest and /mana/consensus/nhighest
 	t.Run("mana/*/nhighest", func(t *testing.T) {
-		expectedAccessOrder := []identity.ID{faucet.ID(), peers[1].ID(), peers[2].ID()}
+		expectedAccessOrder := []identity.ID{faucet.ID(), peers[1].ID()}
```

Comment:
Limited access order list to two nodes only, because third and fourth node have roughly the same amount of mana, which causes them to sometimes be returned in different order.